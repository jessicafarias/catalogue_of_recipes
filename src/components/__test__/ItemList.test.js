import { BrowserRouter } from 'react-router-dom';
import React from 'react';
import '@testing-library/jest-dom';
import TestRenderer from 'react-test-renderer';
import ListItem from '../listItem';

describe('Display ItemList.js', () => {
  it('renders correctly', () => {
    const tree = TestRenderer.create(
    <BrowserRouter>
      <ListItem id={1} order={0}/>
    </BrowserRouter>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});