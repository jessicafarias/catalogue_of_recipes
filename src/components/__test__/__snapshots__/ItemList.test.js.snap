// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display ItemList.js renders correctly 1`] = `
<li
  className="active"
  data-slide-to={0}
  data-target="#slide_in1"
/>
`;
